src/tic_tac_toe/GameBoard.java

```diff
@@ -32,8 +32,11 @@ public void playingGame() {
             printBoard();
             System.out.println("Hey user, pick a number between 1-9");
```

Comment:
This playingGame method is getting pretty big. Not a biggie for now since this is a basic tic tac toe program. But just for future reference, you want to keep your methods as concise as you can. Each method should theoretically do 'one' thing. Typically when my methods gets too big, i take chunks of the logic in the method and put them in a new method and just call that new method in the original method. Makes it easier down the line to debug if theres an issue. As well as easier to maintain.

Reply:
Good to know, that would probably make things alot easier. I'll do that from now on.